pretyping/inferCumulativity.mli

```diff
@@ -0,0 +1,10 @@
+(************************************************************************)
+(*  v      *   The Coq Proof Assistant  /  The Coq Development Team     *)
+(* <O___,, *   INRIA - CNRS - LIX - LRI - PPS - Copyright 1999-2017     *)
+(*   \VV/  **************************************************************)
+(*    //   *      This file is distributed under the terms of the       *)
+(*         *       GNU Lesser General Public License Version 2.1        *)
+(************************************************************************)
+
+val infer_inductive : Environ.env -> Entries.mutual_inductive_entry ->
```

Comment:
@SkySkimmer: just out of curiosity, why don't you need an evar map here?

Reply:
- there are no evars (we are after they get substituted with their definitions)
- local universes are in the `mutual_inductive_entry`, and we don't modify them (we need to know which ones they are for the analysis)

Since those are the 2 things evar maps are for there is no need for it.

(We also push local universes into the environment, in part because the previous infer system needed that, but I think now we could get away with skipping that step since we don't look at the ugraph. Let's leave that for a later PR.)